pdc/apps/utils/utils.py

```diff
@@ -59,3 +61,43 @@ def read_permission_for_all():
 
 def get_model_name_from_obj_or_cls(obj_or_cls):
     return ContentType.objects.get_for_model(obj_or_cls).model
+
+
+class RelatedNestedOrderingFilter(OrderingFilter):
+    """
+    Extends OrderingFilter to support ordering by fields in related models
+    using Django orm '__'.
+
+    This class is copied and changed from the patch of the github issue.
+    https://github.com/encode/django-rest-framework/issues/1005
```

Comment:
OK, now I get it. Can you use the updated code from [the latest comment](https://github.com/encode/django-rest-framework/issues/1005#issuecomment-289555282) and mention the comment link in the docstring instead?